changelog/888.doc.rst

```diff
@@ -0,0 +1 @@
+Add missing description to ``dropdown.py`` example.
```

Comment:
pretty sure we don't need a changelog entry for this pr

Reply:
how will the check pass then

Reply:
skip news label

Reply:
75a7811e2a43aa23657218f2b587418729ed2889